operator/roles/kiali-deploy/tasks/openshift/os-main.yml

```diff
@@ -65,5 +65,9 @@
 - name: Process OpenShift Console Links
   k8s:
     definition: "{{ lookup('template', 'templates/openshift/console-links.yaml') }}"
+  vars:
+    # For some reason, in some cases when the operator runs, the kiali.io/member-of tag is not set yet.
+    # Maistra-ones are always present however.
+    namespaces: "{{ lookup('k8s', api_version='v1', kind='Namespace', label_selector=('maistra.io/member-of=' + kiali_vars.deployment.namespace)) | default({}) | json_query('[].metadata.name') if '**' in all_accessible_namespaces else all_accessible_namespaces }}"
```

Comment:
This is fine by me - it just means we need to say we only support ConsoleLinks when running with ServiceMesh. But I don't see a problem with this - the only time you will run into a non-supported configuration is if you are on OpenShift, *and* you are using upstream Istio... but that will rarely, if ever, happen since ServiceMesh is shipping with the latest OpenShift releases so virtually everyone will use it and not upstream Istio.